scripts/build_shapes.py

```diff
@@ -1379,4 +1429,10 @@ def gadm(
         nprocesses=nprocesses,
         simplify_gadm=simplify_gadm,
     )
+
+    if snakemake.params.get("subregion").get("define_by_gadm", False):
+        subregion_config = snakemake.params.subregion["define_by_gadm"]
+        subregion_shapes = crop_country(gadm_shapes, subregion_config)
+        subregion_shapes.to_file(snakemake.output.subregion_shapes)
```

Comment:
It would be nice to always have the subregion shape in the workflow to simplify the snakefile. If so, the use of save_to_geojson file could be nice: `save_to_geojson(gpd.GeoDataFrame(), snakemake.output.subregion_shapes)` should save an empty file